docs/config/README.md

```diff
@@ -1683,7 +1683,7 @@ To enable it, set `disabled` to `false` in your configuration file.
 
 [git_metrics]
 added_style = "bold blue"
-format = '[+$added]($added_style)/[-$deleted]($deleted_style) '
+format = "[+$added]($added_style)/[-$deleted]($deleted_style)"
```

Comment:
```suggestion
format = "[+$added]($added_style)/[-$deleted]($deleted_style) "
```

Reply:
Thank you very much. Please let me know if I need to combine all the commits into one.